packages/expo-cli/src/commands/permissions.ts

```diff
@@ -0,0 +1,416 @@
+      if (key in infoPlist && infoPlist[key]) {
+        currentDescriptions[key] = infoPlist[key];
+      } else {
+        currentDescriptions[key] = '';
+      }
+    }
+  } else {
+    if ((exp.ios || {}).infoPlist) {
+      console.log(
+        chalk.magenta(
+          `${CHEVRON} Getting permissions from the ${chalk.underline`expo.ios.infoPlist`} object in the universal ${chalk.bold`app.json`}...`
+        )
+      );
+    } else {
+      console.log(
+        chalk.magenta(
+          `${CHEVRON} Showing the default permissions used by ${chalk.bold`Turtle`} to build your project...`
+        )
+      );
+    }
+    defaultExpoDescriptions = Object.keys(DefaultiOSPermissions).reduce(
+      (previous, current) => ({
+        ...previous,
+        [current]: getDefaultExpoPermissionDescription(appName, current),
+      }),
+      {}
+    );
+    currentDescriptions = Object.keys(DefaultiOSPermissions).reduce(
+      (previous, current) => ({
+        ...previous,
+        [current]: getPermissionDescription(appName, exp, current),
+      }),
+      {}
+    );
+  }
+
+  const choices: IOSPermissionChoice[] = [];
+  for (const key of Object.keys(currentDescriptions)) {
+    choices.push({
+      name: key,
+      message: DefaultiOSPermissionNames[key],
+      initial: currentDescriptions[key],
+    });
+  }
+
+  const answer = await promptForPermissionDescriptionsAsync(
+    choices,
+    !!supportingDirectory,
+    currentDescriptions
+  );
+
+  const modifiedAnswers: { [key: string]: string | undefined } = Object.keys(answer).reduce(
+    (previous, current) => {
+      const permissionDescription = answer[current];
+
+      if (permissionDescription !== defaultExpoDescriptions[current] && permissionDescription) {
+        return {
+          ...previous,
+          [current]: permissionDescription,
+        };
+      }
+      return {
+        ...previous,
+        [current]: undefined,
+      };
+    },
+    {}
+  );
+
+  await writePermissionsToIOSAsync(projectDir, exp, modifiedAnswers);
+
+  if (supportingDirectory && infoPlist) {
+    console.log(
+      chalk.magenta(`${CHEVRON} Saving selection to the native ${chalk.bold`Info.plist`}...`)
+    );
+    await IosPlist.modifyAsync(supportingDirectory, 'Info', infoPlist => {
+      for (const key of Object.keys(answer)) {
+        if (answer[key]) {
+          infoPlist[key] = answer[key];
+        } else {
+          delete infoPlist[key];
+        }
+      }
+      return infoPlist;
+    });
+  } else {
+    console.log(
+      `${CHEVRON} ${chalk.bold`Remember:`} ${chalk.reset
+        .dim`Permission messages are a build-time configuration. Your selection will only be available in Standalone, or native builds. You will continue to see the predefined permission dialogs when using your app in the Expo client.`}`
+    );
+  }
+}
+
+// Find the location of the native iOS info.plist if it exists
+// TODO(Bacon): Search better
```

Comment:
explain what you mean by this - someone reading this comment is not going to know what to do